src/Exception.php

```diff
@@ -1,14 +0,0 @@
-<?php
-
-namespace PodPoint\Payments;
-
-class Exception extends \Exception
-{
-    /**
-     * @param \Throwable|null $previous
-     */
-    public function __construct(\Throwable $previous = null)
-    {
-        parent::__construct('Failed to create payment due to provider exception', 1, $previous);
-    }
```

Comment:
I have an idea on how we can approach exceptions in this project, I'll create a PR from this branch to show you as I think that's the easiest way

Reply:
cool